docat/Makefile

```diff
@@ -0,0 +1,7 @@
+format:
```

Comment:
Do we need he makefile if we dont use it in e.g. docke

Reply:
yes.

Reply:
For wat?

Reply:
fun and because i am lazy